tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivativeImpl.tpp

```diff
@@ -1750,17 +1752,26 @@ void test_impl(const Spectral::Quadrature quadrature,
             std::make_pair(local_direction, local_neighbor_id);
         const auto& mortar_mesh = mortar_meshes.at(mortar_id);
         const auto& mortar_size = mortar_sizes.at(mortar_id);
-        auto boundary_data_on_mortar =
-            Spectral::needs_projection(face_mesh, mortar_mesh, mortar_size)
-                ? ::dg::project_to_mortar(packaged_data, face_mesh, mortar_mesh,
-                                          mortar_size)
-                : std::move(packaged_data);
-
-        std::vector<double> expected_data{
-            boundary_data_on_mortar.data(),
-            boundary_data_on_mortar.data() + boundary_data_on_mortar.size()};
+
+        DataVector expected_data{};
+        if (Spectral::needs_projection(face_mesh, mortar_mesh, mortar_size)) {
+          expected_data = DataVector{
+              mortar_mesh.number_of_grid_points() *
+              Variables<mortar_tags_list>::number_of_independent_components};
+          Variables<mortar_tags_list> projected_packaged_data{
+              expected_data.data(), expected_data.size()};
+          const auto blah = ::dg::project_to_mortar(packaged_data, face_mesh,
```

Comment:
maybe rename `blah` to something a bit clearer? I think you did this in `src` already

Reply:
Oops! Yeah forgot to change that. I just copied what I did in `InternalMortarDataImpl.hpp` (also copied the comment that went along with it for clarity)